partials/tabs/projects.html

```diff
@@ -68,7 +68,7 @@
         <div ng-show="currentProject.difficulty" class="project-detail-element">
           <div class="small-heading uppercase">Difficulty</div> <span class="pr-element-detail chip">{{ currentProject.difficulty }}</span> </div>
         <div ng-show="currentProject.mentors.length>0" class="project-detail-element">
-          <div class="small-heading uppercase">Mentors</div> <span class="pr-element-detail chip" ng-repeat="mentor in currentProject.mentors">@{{mentor}}</span> </div>
+          <div class="small-heading uppercase">Mentors</div> <span class="pr-element-detail chip" ng-repeat="mentor in currentProject.mentors"><a ng-href="https://github.com/{{mentor}}" class="mentors-github-id chip clickable" target="_blank">@{{mentor}}</a></span> </div>
```

Comment:
1. There is no use of class `clickable` here as it just sets the cursor pointer. Since an anchor tag has been used, adding `clickable` class is just adding an extra CSS that can be avoided.
2. Break the HTML code to multiple lines for this `div` as it will look pretty. It will be good if you start anchor tag from next line ;-)